### DISCUSSION TITLE
A first party GitHub OAuth has been added to my account.... it seems impossable to talk to this person LOL<title>

### DISCUSSION THREAD

**Discussion by MarkTscater** [community/community#39638] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have had the worst time in my life trying to figure out how to tie my address to a smart contract and have Etherscan accept it on my profile of verified addresses.....(trying on and off for 3 and half years, now full court press last 6 months all ending in failure.....)

it would be nice if this GitHub OAthth person could talk to me one on one somehow... but it seems like brain surgery to find out how to do this.....

I am sorry to bitch about this publicly... but how can i get a hold of this new OAuth person that has been add to my account?

when i try to find a way to talk to this person directly I am confronted by a giant list of stuff that honestly I don't comprehend....

**Comment by MarkTscater** [comment]
To the person Linuxlawson, i am laughing also dude... just tried to find you in the people search... but your not there.... maybe you would know enough to help me out.....

**Reply by radar-blip** [reply]
No sorry, I dont know enough to help you out with that. Your post just sounded kinda funny. 
3 and a half years???
Anyway, Good luck with all that, wish I could help!
Have a nice flight!

**Reply by MarkTscater** [reply]
Amazing that you are the only one to see this or commit on it... lol these big companies , I swear any idea how to get anyone else to see this?

**Reply by radar-blip** [reply]
Hold please.

**Reply by radar-blip** [reply]
Sorry, I can't find any helpful info for you.

**Comment by djdefi** [comment]
OAuth is an authentication protocol, not a person. 

For information on building OAuth apps, the following guide may be useful: https://docs.github.com/en/developers/apps/building-oauth-apps

**Comment by simoncfc** [comment]
Hi you fat man in your mums basement. Do you really think people will fall
for this you’ve not even hidden the other emails you’ve sent this too which
is scammer 101.

Pathetic attempt.

On Wed, 28 Dec 2022 at 23:28, SusanThompson4639 ***@***.***>
wrote:

> Already wet from dirty thoughts bout you https://4.fo/pbTsQzT
> @stfDeveloper <https://github.com/stfDeveloper> @SimoMassaro52
> <https://github.com/SimoMassaro52> @Hamza-bouzid
> <https://github.com/Hamza-bouzid> @Vito9009 <https://github.com/Vito9009>
> @Mosi8 <https://github.com/Mosi8> @matteopompei
> <https://github.com/matteopompei> @cybernard0
> <https://github.com/cybernard0> @MyLastWords
> <https://github.com/MyLastWords> @tomitou69 <https://github.com/tomitou69>
> @ziednamouchi <https://github.com/ziednamouchi> @Joker666666
> <https://github.com/Joker666666> @jdebetaz <https://github.com/jdebetaz>
> @akacodeorg <https://github.com/akacodeorg> @neocollin
> <https://github.com/neocollin> @EchoCharlie-07
> <https://github.com/EchoCharlie-07> @elmaaradi
> <https://github.com/elmaaradi> @RWentworth <https://github.com/RWentworth>
> @andetero <https://github.com/andetero> @Gabrielerusso
> <https://github.com/Gabrielerusso> @dyegocaldeira
> <https://github.com/dyegocaldeira> @0mniteck <https://github.com/0mniteck>
> @anon-one <https://github.com/anon-one> @riccjohn
> <https://github.com/riccjohn> @Locathelli <https://github.com/Locathelli>
> @renton61 <https://github.com/renton61> @NitescuLucian
> <https://github.com/NitescuLucian> @rsch4dfx <https://github.com/rsch4dfx>
> @hibare <https://github.com/hibare> @grantinov
> <https://github.com/grantinov> @duhfreakinduh
> <https://github.com/duhfreakinduh> @0p71mu5 <https://github.com/0p71mu5>
> @shangeth <https://github.com/shangeth> @hristog95
> <https://github.com/hristog95> @Smackdaddy66
> <https://github.com/Smackdaddy66> @eknowles <https://github.com/eknowles>
> @simoncfc <https://github.com/simoncfc> @Briteless
> <https://github.com/Briteless> @andrew-oko-odion
> <https://github.com/andrew-oko-odion> @Scr1ptz
> <https://github.com/Scr1ptz> @Valkhiry <https://github.com/Valkhiry>
> @kaskoo <https://github.com/kaskoo> @hdytyldrm
> <https://github.com/hdytyldrm> @basarik <https://github.com/basarik>
> @cenkgun <https://github.com/cenkgun> @dhruba1986
> <https://github.com/dhruba1986> @halilozcan
> <https://github.com/halilozcan> @monowar1993
> <https://github.com/monowar1993> @ankityadaw0
> <https://github.com/ankityadaw0> @canergulgec
> <https://github.com/canergulgec> @samykhelifa
> <https://github.com/samykhelifa>
> Hi wanna see my nude photo https://4.fo/pbTsQzT
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/39638#discussioncomment-4513452>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AH7WCV4U6LPMW22OZQSIIFLWPTEIVANCNFSM6AAAAAASGLVOCM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
Regards Simon Reynolds
